File: owasp01.injection/Controllers/HomeController.vb
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        Return View()
    End Function

    Function About() As ActionResult
        ViewData("Message") = "OWASP Top 10 - 01. Injection"

        Return View()
    End Function

    Function Contact() As ActionResult
        ViewData("Message") = "Contact the author"

        Return View()
    End Function
End Class

File: owasp4net.injection/Controllers/HomeController.cs
using Microsoft.AspNetCore.Mvc;

namespace owasp4net.injection.Controllers
{
    public class HomeController : Controller
    {
        public IActionResult Index()
        {
            return View();
        }
        
        public IActionResult About()
        {
            ViewData["Message"] = "About OWASP 4 .NET";

            return View();
        }

        public IActionResult Contact()
        {
            ViewData["Message"] = "Contact page.";

            return View();
        }        
    }
}